etc/config/compiler-explorer.amazon.properties

```diff
@@ -15,15 +15,6 @@ proxyRetries=10
 proxyRetryMs=500
 rescanCompilerSecs=3600
 sentryDsn=https://8e4614f649ad4e3faf3e7e8827b935f9@sentry.io/102028
-formatters=clangformat:rustfmt
-formatter.clangformat.name=clang-format
-formatter.clangformat.exe=/opt/compiler-explorer/clang-trunk/bin/clang-format
-formatter.clangformat.styles=Google:LLVM:Mozilla:Chromium:WebKit:Microsoft:GNU
-formatter.clangformat.type=clangformat
-formatter.rustfmt.name=rustfmt
-formatter.rustfmt.exe=/opt/compiler-explorer/rustfmt-1.4.36/rustfmt
-formatter.rustfmt.styles=
-formatter.rustfmt.type=rustfmt
```

Comment:
I think we want to move this out of compiler-explorer.properties, similar to execution.properties.

For now, I've moved the category down to make it easier to read the other properties. Nonfunctional change.

Reply:
Agreed; this is getting quite messy. Perhaps it needs to be per-language?

Reply:
That could work, maybe something like this 
```properties
formatters=cpp:c:rust:go
formatter.cpp.name=clang-format
formatter.cpp.exe=/opt/compiler-explorer/clang-trunk/bin/clang-format
formatter.cpp.styles=Google:LLVM:Mozilla:Chromium:WebKit:Microsoft:GNU
formatter.cpp.type=clangformat
... etc
```

Reply:
Putting it into the language config makes sense, but I figured having it per-language; the example you posted above seemed to list all the formatters in one file though!  I was thinking `cpp.defaults.properties` would have its own `formatters` etc

Reply:
That sounds good to me. I'd prefer to commit that as a separate patch because it's a decent-size surgery that is not necessarily related to gofmt. How does that sound?

Reply:
Sounds perfect!